index.js

```diff
@@ -0,0 +1,36 @@
+$(document).ready(function() {
+	console.log('if i cant even get this right i dont know why im trying');
+
+
+	$('a').click(function(event) {
+		console.log('maybe if i get here ill feel a lil better');
+		event.preventDefault();
```

Comment:
This is saying that all links on your page will not function as links. The function `event.preventDefault` tells whatever selector you are targeting to **not** perform its intended functionality. This could get sticky if there are actual links on the page that need to go to other pages/sites. You should have included `event.preventDefault` for each function for the '.readmore' or '.readless' click events.